client/blocks/login/style.scss

```diff
@@ -26,9 +31,25 @@
 }
 
 .login__form-remember-me {
-	margin: 5px 0 15px;
+	margin-bottom: 20px;
 
 	input {
 		margin-right: 10px;
 	}
-}
\ No newline at end of file
+	label{
+		font-size: 14px;
+	}
+}
+
+.wp-login__footer{
```

Comment:
Just wondering why we are redefining the styles for `.wp-login__footer` here?
Isn't it a class used in the parent component `wp-login`? Also since all styles are merged into one file currently it shouldn't be any different than having those style changes in `client/login/wp-login/style.scss`.